Formula/k/kubetail.rb

```diff
@@ -0,0 +1,38 @@
+# typed: false
+# frozen_string_literal: true
+
+# This file was generated by GoReleaser. DO NOT EDIT.
+class Kubetail < Formula
+  desc "CLI tool with a web dashboard for viewing Kubernetes logs"
+  homepage "https://www.kubetail.com/"
+  url "https://github.com/kubetail-org/kubetail/archive/refs/tags/0.8.0-rc2.tar.gz"
+  version "0.0.1"
```

Comment:
if you use stable release, you dont need this version override.

Reply:
Used stable release and removed version override